cmd/kots/cli/admin-console-push-images.go

```diff
@@ -131,7 +131,9 @@ func getECRLogin(endpoint string, keyID string, accessKey string) (string, strin
 
 func getECRService(accessKeyID, secretAccessKey, zone string) *ecr.ECR {
 	awsConfig := &aws.Config{Region: aws.String(zone)}
-	awsConfig.Credentials = credentials.NewStaticCredentials(accessKeyID, secretAccessKey, "")
+	if (accessKeyID != "" && secretAccessKey != "") {
```

Comment:
ditto
```suggestion
	if accessKeyID != "" && secretAccessKey != "" {
```

Reply:
Thanks - I've been working in JS too long 😄